src/plugins/gui/fiatPlugin.tsx

```diff
@@ -78,21 +85,51 @@ export const executePlugin = async (params: {
     },
     popScene: async () => {
       navigation.pop()
+    },
+    enterFieldsForm: async (params: FiatPluginSepaFormParams) => {
+      return new Promise((resolve, reject) => {
+        const { countryCode, forms, headerIconUri, headerTitle, onSubmit } = params
+        navigation.navigate('guiPluginEnterForm', {
+          countryCode,
+          headerIconUri,
+          headerTitle,
+          forms,
+          onSubmit: async (fieldInputs: FormFieldProps[]) => {
+            await onSubmit(fieldInputs)
+            resolve(fieldInputs)
+          }
+        })
+      })
+    },
+
+    transferInfo: async (params: FiatPluginSepaTransferParams) => {
+      const { headerTitle, groupedDisplayData, promptMessage, onSubmit } = params
+      return new Promise((resolve, reject) => {
+        navigation.navigate('guiPluginSepaTransfer', {
+          headerTitle,
+          groupedDisplayData,
+          promptMessage,
+          onSubmit: async () => {
+            await onSubmit()
+            resolve()
+          }
+        })
+      })
     }
   }
 
   if (guiPlugin.nativePlugin == null) {
     throw new Error('executePlugin: missing nativePlugin')
   }
 
-  const plugin = await guiPlugin.nativePlugin({ disablePlugins, showUi, account })
+  const plugin = await guiPlugin.nativePlugin({ disablePlugins, showUi, account, state, dispatch })
```

Comment:
No. No. Can't pass `state` and `dispatch` to a plugin. Plugins are pure non-gui code with no access to redux